lm_eval/models/sg_llm.py

```diff
@@ -22,7 +33,7 @@ def __init__(
         batch_size: int,
         no_noise: bool = False,
         bfloat_16: bool = False,
-        load_in_8bit: bool = False,
+        quantization_type: QT = QuantizationType.default,
```

Comment:
There is no reason to use a `TypeVar` if this isn't a generic class.

```suggestion
        quantization_type: QuantizationType = QuantizationType.default,
```